tests/distributions/test_transforms.py

```diff
@@ -327,3 +328,40 @@ def test_sylvester(self):
     def test_normalize_transform(self):
         for p in [1., 2.]:
             self._test(lambda p: T.Normalize(p=p), autodiff=False)
+
+
+@pytest.mark.parametrize('batch_shape', [(), (7,), (6, 5)])
+@pytest.mark.parametrize('dim', [2, 3, 5])
+@pytest.mark.parametrize('domain', [dist.constraints.corr_cholesky_constraint,
+                                    dist.constraints.lower_cholesky])
+def test_inv_cholesky_transform(batch_shape, dim, domain):
```

Comment:
Is there a reason you're not adding a standard test `TestTransforms.test_inv_cholesky_transform`? We've put a lot of work into those standardized tests, and it's a shame to not exercise them.

Reply:
I think we can add another kind of Flatten to test the jacobian of those special matrices. Let me figure it out how to merge this jacobian test to the current TestTransforms.

Reply:
Note there's a `Reshape` transform in the latest PyTorch, in case that helps. We could temporarily port that to Pyro, then remove it next month.

Reply:
I would argue that those standardized test, with assumptions that the domain/codomain is real or real_vector, makes it pretty tricky to test for those positive_definite, corr_matrix domains. I can use some composed transforms to make tests pass but would also like to have separate tests that are more intuitive.

Reply:
@fritzo I made several attempts but we need to use the standard tests for something like `ComposeTransform([LowerCholeskyTransform(), CholeskyTransform().inv, CholeskyTransform()])`. But it is not clear to me that composing `t.inv` and `t` will actually assert that both `t._call` and `t._inverse` give correct shapes. I would like to accept the shame and have different tests for those cholesky transforms.

Reply:
@fehiepsi Sounds good thanks for attempting! I think this is valuable feedback for @stefanwebb and @feynmanliang as they create tests for https://github.com/stefanwebb/flowtorch